Update with latest Design

Adnanarodiya opened the pull request.

max-programming commented:
Thanks for the updates @Adnanarodiya. Can you make the following updates?


1. The `/repos/[languageId]` pages are throwing 404 page on prod. Can you look into that?

1. Color updates on 404 page
<img width="975" height="416" alt="image" src="https://github.com/user-attachments/assets/ba0a0a59-1027-455f-b05c-d204c10d68ff" />

max-programming reviewed (changes requested):
Thanks for the updates @Adnanarodiya. Can you make the following updates?


1. The `/repos/[languageId]` pages are throwing 404 page on prod. Can you look into that?

2. Add some click effect on the buttons

3. Change the font `Space_Grotesk` to `Atkinson_Hyperlegible_Mono`

1. Color updates on 404 page
<img width="975" height="416" alt="image" src="https://github.com/user-attachments/assets/ba0a0a59-1027-455f-b05c-d204c10d68ff" />

1. Images on contributors page are too zoomed in 
<img width="396" height="434" alt="image" src="https://github.com/user-attachments/assets/93e09b5d-b583-4695-9337-949e4b273e7d" />

Adnanarodiya commented:
Okay ill update

max-programming commented:
@Adnanarodiya I updated your fork with latest code from here
Just `pull` and `pnpm install` before making any changes

Adnanarodiya commented:
Okay i'll update the changes.

Adnanarodiya commented:
###I've implemented all the requested changes:

1. Fixed the /repos/[languageId] 404 issue.
2. Added click effects to buttons.
3. Updated the font to Atkinson_Hyperlegible_Mono.
4. Improved 404 page colors.
5. Adjusted contributor images to proper scale.
6. Enhanced heading contrast and button styles.
7. Updated report modal to match new styles.
8. Updated loading.tsx with the new skeleton loader colors.
9. Let me know if anything else needs to be adjusted!

max-programming commented:
Thanks @Adnanarodiya

This was a wild ride of breaking changes with the new upgraded libraries
